src/Features/Core/Portable/Completion/Providers/AbstractRecommendationServiceBasedCompletionProvider.cs

```diff
@@ -47,6 +48,13 @@ internal abstract class AbstractRecommendationServiceBasedCompletionProvider<TSy
                 // Just filter valid symbols. Nothing to preselect
                 return recommendedSymbols.NamedSymbols.SelectAsArray(IsValidForGenericConstraintContext, s => (s, preselect: false));
             }
+            else if (context.IsInheritanceContext)
+            {
+                Debug.Assert(context.DeclarationOfInheritingSymbol is not null, $"{nameof(context.DeclarationOfInheritingSymbol)} must never be null here");
```

Comment:
i think there can be an attribute on IsInheritanceContext/DeclarationOfInheritingSymbol  to enforce this.

Reply:
Can you pleasee tell what exact attribute it is? Nullability is not my strongest thing 🙁